include/jemalloc/internal/jemalloc_internal_types.h.in

```diff
@@ -33,17 +33,18 @@ typedef enum malloc_init_e malloc_init_t;
  *
  * a: arena
  * t: tcache
+ * .: configuration-dependent
```

Comment:
Suggest calling out the name of the config param which controls this.

Reply:
Done